internal/cli/orgs.go

```diff
@@ -66,3 +67,25 @@ var CmdOrgCreate = &cobra.Command{
 	SilenceErrors: true,
 	SilenceUsage:  true,
 }
+
+// CmdOrgDelete implements the epinio `orgs delete` command
+var CmdOrgDelete = &cobra.Command{
+	Use:   "delete NAME",
+	Short: "Deletes an organization",
+	Args:  cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
This is a dangerous spell! Should we maybe show a warning and require confirmation? E.g.

```
You are about to delete organization "myorg" and everything included in it (applications, services etc). Are you sure? (Y/n)
```

Reply:
Right, will add this.

Reply:
Done.